### DISCUSSION TITLE
CoCalc and ResearchRabbit

### DISCUSSION THREAD

**Discussion by nikkujensen** [sagemathinc/cocalc#6181] [General]
I recently started using CoCalc for research. I also use the tool ResearchRabbit. Are there any ideas of how these two could be integrated? ResearchRabbit is simply a webpage with login credentials
https://www.researchrabbit.ai/

**Comment by williamstein** [comment]
Can you explain how you're using ReseachRabbit and what you would get out of some sort of integration?

**Comment by haraldschilly** [comment]
@IloveJesusmyKing in general, since those are different websites by different organizations, what would be needed is an API and a description of what you want to accomplish. Just "integrated" is way too vague.

As a first step: if there is an API for research rabbit accessible via the internet, you can use it from within a CoCalc project – just make sure to upgrade the project to have internet access.

**Comment by nikkujensen** [comment]
Good points @haraldschilly @williamstein 

My thought, at least one of them, is this. That research rabbit could pull references right into a bib file in a CoCalc project. This feature is similar to what research rabbit offers for Zotero already. [Here is an example](https://youtu.be/6vVcqwdpfK0?t=399). However, in that instance, I think Zotero is the one with the API and research rabbit pushes citations/information that way.

Any thoughts regarding this?
